sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ec2/files/manglefs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+to_delete=(
+  /etc/systemd/system/amazon-ssm-agent.service
+  /etc/systemd/system/multi-user.target.requires/coreos-metadata-sshkeys@core.service
+  /etc/amazon/ssm/
+  /etc/eks/bootstrap.sh
+)
+
+rm -rf "${to_delete[@]/#/${rootfs}}"
```

Comment:
I don't understand this line, what creates the files under `/etc`?

Reply:
The list of old OEM files should now go to the misc-files package: https://github.com/flatcar/scripts/pull/1016
We should boot an instance and check the old contents of `/oem/`.

Reply:
So, the above can be deleted here, or?